Cargo.toml

```
@@ -26,6 +26,9 @@ debug = true
 debug = "line-tables-only"
 
 [workspace.dependencies]
+delta_kernel = { version = "0.1" }
+# delta_kernel = { path = "../delta-kernel-rs/kernel" }
```

```suggestion
delta_kernel = { version = "0.1", path = "../delta-kernel-rs/kernel" }
```

These can combined for ease of development, since [cargo allows multiple locations](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations)